src/main/java/com/CoachPlan/service/CoachServiceStub.java

```diff
@@ -14,4 +14,10 @@ public CoachDTO loginByID(String Id) {
 		return coachDTO;
 	}
 
+	@Override
+	public CoachDTO loginByPass(String password) {
+		// TODO Auto-generated method stub
+		return null;
+	}
```

Comment:
This looks like a partially implemented feature.  I wouldn't add this until the feature is fully implemented.  If this branch is merged, it will be merged into master, and only complete features should be merged to master.  Incomplete features should remain in the feature branch, until the feature branch meets the definition of done.

As it is, if someone invokes this, thinking they will get a CoachDTO object in return, they will actually get a null instead, which will likely end up with a NullPointerException that will crash the program.